docs/result.md

```diff
@@ -227,3 +227,35 @@ File: document.json
       }
     }
 }
+```
+##Client report failure
+
+a POST request is invoked by the client to the server. It returns a JSON file with information like client_id, job_id, and Massages to return failure endpoint in the server. The client can return a failure do to many reasons such as:
+* 400: Bad request
+* 401: Unauthorized
+* 404: Not found
+* 408: Request Timeout
+* 480: Temporarily Unavailable
+
+
+## JSON file definition
+
+* `client_id`: A unique identifier of the client returning the result.
+* `job_id`: ID of the current job object.
+* `Massages`: a string list contains description of the error
+
+## Examples for failure jobs
+
+* 400: Bad request
+  * Incorrect URL:
+
+    Incorrect_URL = "https://api.data.gov:443/requlations/v3/download.json?"\"documentID=EPA-HQ-OAR-2011-0000-0108&contentType=pdf"
+
+Example of JSON file return to return_failure endpoint 
+```
+    {
+      'client_id': 'client14',
+      'job_id': 'job33',
+      'Masg': '400: Bad request'
```

Comment:
The message should include more detailed information so that we can understand *why* things fail.  For example, If it is a bad URL, we should include the URL.